index.html

```diff
@@ -7,22 +7,20 @@
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
     <link href="https://fonts.googleapis.com/css2?family=Inter:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="style.css">
+    <script src="JS/script.js"></script>
     <title>Document</title>
 </head>
 <body>
-
     <section class="container">
-        <nav class="NavBar"> 
-            <div class="link">
-                <a href="">Portfolio</a>
-                <a href="">About</a>
-                <a href="">Contact</a>
-            </div>
-            
-            <div class="menu">
-                <a href="" class="fa fa-bars"></a>
-            </div>     
-        </nav>  
+        <nav id="navBar" class="NavBar"> 
+            <a href="#cards">Portfolio</a>
+            <a href="#About">About</a> 
+            <a href="#contForm">Contact</a>  
+        </nav>
+        
+        <div id="menu" class="menu">
+            <a href="javascript:void(0)" onclick="hello()" class="fa fa-bars"></a>
```

Comment:
- I noticed when i click on the hamburger menu, it has a background color, please kindly take off the hover effect as that's what gives it the color, and try to follow the Figma design, so you have a similar-looking design as much as possible. 

##### Your design 
![image](https://github.com/Dachrono/Andy_Portfolio/assets/58771507/72b8b4b3-36fa-46fb-90db-1df028d02c83)

##### Figm design 
![image](https://github.com/Dachrono/Andy_Portfolio/assets/58771507/78b19f59-6a75-41df-958c-2662d716f94d)

Reply:
Solved